.github/workflows/ci.yml

```diff
@@ -54,6 +54,7 @@ jobs:
         python-version: '3.x'
     - name: Install dependencies
       run: |
+        sudo apt install python3-dev
```

Comment:
why do we need this?

Reply:
That was a flawed test that I was doing at ci. Already removed!